gum/gum.c

```diff
@@ -506,6 +509,18 @@ gum_on_log_message (const gchar * log_domain,
 
     return;
   }
+#else
+# ifdef HAVE_WINDOWS
+  gunichar2* utf16_message;
+  utf16_message = g_utf8_to_utf16 (message, -1, NULL, NULL, NULL);
+  if (utf16_message)
```

Comment:
No need to check the result, we are in control of the call-sites and can assume they give us valid UTF-8.